openpype/hosts/maya/lib.py

```diff
@@ -1,26 +1,21 @@
 import os
-import shutil
+from openpype.settings import get_project_settings
 
 
-def copy_workspace_mel(workdir):
-    # Check that source mel exists
-    current_dir = os.path.dirname(os.path.abspath(__file__))
-    src_filepath = os.path.join(current_dir, "resources", "workspace.mel")
-    if not os.path.exists(src_filepath):
-        print("Source mel file does not exist. {}".format(src_filepath))
-        return
-
-    # Skip if workspace.mel already exists
+def create_workspace_mel(workdir, project_name):
     dst_filepath = os.path.join(workdir, "workspace.mel")
     if os.path.exists(dst_filepath):
         return
 
-    # Create workdir if does not exists yet
     if not os.path.exists(workdir):
         os.makedirs(workdir)
 
-    # Copy file
-    print("Copying workspace mel \"{}\" -> \"{}\"".format(
-        src_filepath, dst_filepath
-    ))
-    shutil.copy(src_filepath, dst_filepath)
+    project_setting = get_project_settings(project_name)
+    mel_script = project_setting["maya"].get("mel_workspace")
+
+    # Skip if mel script in settings is empty
+    if not mel_script:
+        return
```

Comment:
Should this maybe do a `log.debug` saying that there's no mel script defined in settings to create? Or do we feel that having no workspace defined (or set up at all) is a reasonable scenario?

Reply:
I agree it could log it. Could help find out possible issues.

@moonyuet can you add the log? Something like:
```
log = Logger.get_logger("create_workspace_mel")
log.debug(
    "File 'workspace.mel' not created. Settings value is empty."
)
```